base/sysimg.jl

```diff
@@ -248,7 +249,6 @@ include("iobuffer.jl")
 
 # strings & printing
 include("intfuncs.jl")
-include("strings/strings.jl")
```

Comment:
Hmm. It's a bit weird for this to need to move all the way down here because of this change...

Reply:
https://github.com/JuliaLang/julia/blob/master/base/sysimg.jl#L238-#L239 call `string` and they are "in between" the old location of `string` (which was above those line in "strings/string.jl") and a new location (which is now included by `"strings/strings.jl"`).

But as a second thought - probably you are right and only loading `strings/substring.jl` should be moved upwards. I will fix this.